use-cases/pricing/code/python/src/utils/api_utils.py

```diff
@@ -40,6 +41,11 @@ def _get_app_credentials(self):
             raise e
         else:
             return s_dict
+   
+    def _set_useragent(self, client):
+        print('Setting user agent')
```

Comment:
@leonardonakagawa Should we move the print after if?

Reply:
+1